### DISCUSSION TITLE
smoosh compaction extending beyond configured time

### DISCUSSION THREAD

**Discussion by ajitpsingh** [apache/couchdb#4762] [Q&A] [answered]
I am using couch 3.3.2 and I am seeing that compaction end time is extending well  beyond the end time of 03:00 - the configured time. Is there any other configuration I have missed that I should have added?

[smoosh.ratio_dbs]
from = 19:00
to = 03:00
strict_window = true
[smoosh.ratio_views]
from = 19:00
to = 03:00
strict_window = true
[smoosh.slack_dbs]
from = 19:00
to = 03:00
strict_window = true
[smoosh.slack_views]
from = 19:00
to = 03:00
strict_window = true
[smoosh.upgrade_dbs]
from = 19:00
to = 03:00
strict_window = true
[smoosh.upgrade_views]
from = 19:00
to = 03:00
strict_window = true

**Comment by nickva** [comment, marked as answer]
How do you tell the end time, by looking in _active_tasks?

The auto-compaction service (smoosh) is enforcing its time window by suspending the Erlang process performing the actual compaction. So the processes may still show up in _active_tasks (I haven't verified it though yet). 

Another idea is that smoosh is using universal time (UTC) https://www.erlang.org/doc/man/calendar#universal_time-0 so try specifying  the time window in UTC, to see if that's the reason.

**Reply by ajitpsingh** [reply]
Yes we are seeing it from the _active_task and we are using the UTC time zone.
